Calibration/HcalCalibAlgos/python/hcalTestThreshold_cff.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+
+from Calibration.HcalCalibAlgos.hcalTestThreshold_cfi import *
+
+from Configuration.Eras.Modifier_run2_ECAL_2017_cff import run2_ECAL_2017
```

Comment:
Just checking: do you really want to modify for `run2_ECAL_2017`? Not `run2_HCAL_2017`?

Reply:
Yes it is for ECAL_2017

Reply:
Thank you @bsunanda 
Then you also confirm that for 2018 it is for "HCAL_2018" instead? (I'm checking only because I don't understand the logic behind it...)

Reply:
These are thresholds for ECAL receipt energy. Ideally there should be modifiers ECAL_2017 and ECAL_2018. Unfortunately there is no modifier ECAL_2018. That is why I have to use the HCAL_2018 modifier. May be a  cleaner solution is to introduce a modifier ECAL_2018 and then use ECAL_2018 rather than HCAL_2018. I shall try to introduce this later.